front/components/assistant/conversation/InputBar.tsx

```diff
@@ -273,8 +273,56 @@ export function AssistantInputBar({
               suppressContentEditableWarning={true}
               onPaste={(e) => {
                 e.preventDefault();
+
+                // Get the plain text.
```

Comment:
?

Reply:
rebase limbo I think, did not touch that part